scrapers/ProducersFun.yml

```diff
@@ -0,0 +1,27 @@
+name: "Producers Fun"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - producersfun.com/video/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="shadow video-details"]/h1/text()
+    scene:
+      Title: $performer
+      Details: //div[@class="shadow video-details"]/p/text()
+      Date:
+        selector: //p[@class="video-date"]/text()[2]
+        replace:
+          - regex: (st|nd|rd|th)\,
+            with: ","
+        parseDate: January 2, 2006
+      Image: //section[@class="top-wrapper"]/div//video/@poster
+      Tags:
+        Name:
+          selector: //p[@class="video-tags"]/a/text()
+      Performers:
+        Name: $performer
```

Comment:
In some scenes the title is actually different than the performer. Given the fact though that no performer scpecific field is present in the scene page i don't think there is a viable alternative.